Garrett-Iannuzzi/final-countdown
# Display paragraph on App component

## What does this PR do?
- Adds style to `App` component 
- Displays description of app
- Adds icon
- Adds animation on 'I' icon 

## How to test?
- Pull down `display-info` branch
- Run `npm run dev`
- Hover over the "I" icon to see it move and display the rules for the app

## Issues:
- This PR closes no issues 
- The board is up to date 

## Screenshots:
![Screen Shot 2020-03-03 at 11 21 26 PM](https://user-images.githubusercontent.com/48968224/75850950-b7a12600-5da5-11ea-8ab3-7967c3f1aab0.png)
 
## Notes:
- When you hover over the "I" icon you can see the animation

Thanks!
